stl/inc/__msvc_string_view.hpp

```diff
@@ -635,7 +635,7 @@ constexpr size_t _Traits_rfind(_In_reads_(_Hay_size) const _Traits_ptr_t<_Traits
             }
 
             if (_Match_try == _Haystack) {
-                break; // at beginning, no more chance for match
+                return static_cast<size_t>(-1); // at beginning, no more chance for match

```

Comment:
It seems odd to me that we're happy to early-return in this function, but we don't do so on line 631.